.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "gutenberg"]
 	path = gutenberg
-	url = ../../WordPress/gutenberg.git
+	url = ../../illusaen/gutenberg.git
```

Comment:
👋 @illusaen I think it would be better to keep this file pointing to `WordPress` folder as it was previously because it should be generic, what do you think about changing it?

Thanks!